packages/botonic-cli/src/constants.ts

```diff
@@ -0,0 +1,17 @@
+export const BOTONIC_NPM_NAMESPACE = '@botonic'
+export const BOTONIC_HOME_DIRNAME = '.botonic'
+export const GLOBAL_CREDS_FILENAME = 'credentials.json'
+export const BOTONIC_PROJECT_PATH = process.cwd()
+export const BOT_CREDS_FILENAME = '.botonic.json'
+export const ANALYTICS_WRITE_KEY = 'YD0jpJHNGW12uhLNbgB4wbdTRQ4Cy1Zu'
+export const TRACKING_EVENTS = {
```

Comment:
Having ALL constants in a "constants.ts" file is like having a single utils.ts file, a "cajon de sastre". See https://docs.google.com/document/d/1-qd2FdXbAe7XEbdTGyACeCs9HRCauCogocfEU-h-n-I/edit#bookmark=id.3fjmm64cwiqc
But since most of them are "important" :+1: I only suggest moving the TRACKING_EVENTS to somewhere closer to the analytics scope.